CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @njhale @tylerslaton @timflannagan @awgreene
```

Comment:
This works for now, but in the future I think we'll want to use a team to handle this; e.g. `@combo-maintainers`.

Reply:
I was under the impression that was a dedicated alias-related file but maybe I'm too influenced by that behavior from the prow world. Seems like you would need to define a team on the github side-of-things and reference that team in the CODEOWNERS file?